cfme/tests/services/test_service_rbac.py

```diff
@@ -113,6 +119,22 @@ def test_service_rbac_service_catalog(appliance, role_user_group, catalog, catal
         service_request.wait_for_request()
         assert service_request.is_succeeded()
 
+    @request.addfinalizer
+    def _finalize():
+        rest_vm = appliance.rest_api.collections.vms.get(
+            name=f"%{catalog_item.prov_data['catalog']['vm_name']}%"
+        )
+        vm = appliance.collections.infra_vms.instantiate(name=rest_vm.name, provider=provider)
+        vm.delete()
```

Comment:
Not sure guys will like `vm.delete`. Usually I am told to use `delete_if_exists` in the finalizers.

Reply:
Yup! I'd prefer `delete_if_exists` here